libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -644,7 +644,7 @@ const serializeHost = (
       return {
         host: {
           connection: {
-            name: "@parameters('$connections')[" + referenceKey + "]['connectionId']",
+            name: "@parameters('$connections')['" + referenceKey + "']['connectionId']",
```

Comment:
This could be written as 
name: `@parameters('$connections')['${referenceKey}']['connectionId']`

Just a headsUp, this might change pretty soon since LA and PA will be using same connectionformat for open api and this might not look exactly same for all hosts.